cmd/image-builder/main.go

```diff
@@ -46,6 +47,7 @@ func main() {
 		if err != nil {
 			panic(err)
 		}
+		defer sentry.Flush(5 * time.Second)
```

Comment:
I'm not sure I understand the purpose of this, if it panics will it still be executed even with the 5 seconds timer?

Reply:
it takes a maximum 5 seconds to flush existing channels before main ends, but might return before the 5 seconds.